### PR TITLE
Added linter and pre-commit and pre-push hooks using Husky

### PR DESCRIPTION
**Pull request by venkateshpachigulla**
Added linter and pre-commit and pre-push hooks using Husky

### CODE REVIEW

File: src/modules/resume/skill-set/variants/advanced/properties.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const properties = {
         name: 'Skill',
         type: propertyTypes.OBJECT,
         required: true,
-        name: {
+        subname: {
```
Comment (ManojSatishkumar) [inline src/modules/resume/skill-set/variants/advanced/properties.js:25]: will this break @Chiraglr 's variant ?

Comment (RisingGeek) [reply src/modules/resume/skill-set/variants/advanced/properties.js:25]: Yes, this breaks @Chiraglr  's variant. He has 2 keys named `name`. @venkateshpachigulla please change it in `render.js` also.

Comment (venkateshpachigulla) [reply src/modules/resume/skill-set/variants/advanced/properties.js:25]: Added

**Comment by ManojSatishkumar** [comment]
You need to use the "husky" hook and add the pre-commit hook in package.json

**Comment by venkateshpachigulla** [comment]
> You need to use the "husky" hook and add the pre-commit hook in package.json
> Or else it won't run lint on every commit.

Added husky file separately and its running for every commit.

**Comment by ManojSatishkumar** [comment]
> > You need to use the "husky" hook and add the pre-commit hook in package.json
> > Or else it won't run lint on every commit.
> 
> Added husky file separately and its running for every commit.

**Pull request merged**